DiscordBot.DataAccess/SheetsServiceRequests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DiscordBot.DataAccess.Models;
+using Google.Apis.Sheets.v4.Data;
+
+namespace DiscordBot.DataAccess
+{
+    internal static class SheetsServiceRequests
```

Comment:
Maybe `SheetsServiceRequestsHelper` makes it a bit more clear what this is for? 🙂

Reply:
Yeah that's a better name 🙂 Renamed in 284f577